pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "ce"
+version = "3.6.1"
+description = "PCIC's Climate Explorer (CE)"
+authors = ["James Hiebert <hiebert@uvic.ca>"]
```

Comment:
Should we add others?

Reply:
Good suggestion! Yes, that metadata hadn't been updated since the first commit years ago.